docs/guide.rst

```diff
@@ -54,6 +54,37 @@ be between 0.0 and 1.0 inclusive. Vertices out of this range in NDC
 will not introduce any errors, but they will be clipped.
 
 
+Communicating array data
+------------------------
+
+The wgpu-py library makes no assumptions about how you store your data.
+In places where you provide data to the API, it can consume any data
+that supports the buffer protocol, which includes ``bytes``,
+``bytearray``, ctypes arrays, and numpy arrays.
```

Comment:
Does it also accept memoryviews as input?

Reply:
Yes. Will add that.